test/src/tests-feature-flags.ts

```diff
@@ -15,6 +16,14 @@ declare global {
     }
 }
 
+function deleteAllCookies() {
```

Comment:
duplicated function? reduce it!

Reply:
Good catch. I'll move it to our test utilities